mock-core-configs/etc/mock/chroot-aliases.cfg

```diff
@@ -0,0 +1,32 @@
+config_opts["no-config"]["epel-8"] = {
+    "/etc/mock/alma+epel-8": {
```

Comment:
Let's not hardcode paths here if we can help it.

Reply:
The /etc/mock is widely hardcoded, unfortunately.  But in this case, it is doable, I think.

Reply:
I would use the basename too. And in code concat it with `config_path`